sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/models/KeyVaultPermission.java

```diff
@@ -8,28 +8,29 @@
 import java.util.List;
 
 /**
- * A class describing allowed and denied actions and data actions of a {@link KeyVaultRoleDefinition}.
+ * A class describing allowed and forbidden actions and data actions of a {@link KeyVaultRoleDefinition}.
  */
 @Immutable
 public final class KeyVaultPermission {
     private final List<String> allowedActions;
-    private final List<String> deniedActions;
-    private final List<String> allowedDataActions;
-    private final List<String> deniedDataActions;
+    private final List<String> notActions;
+    private final List<KeyVaultDataAction> allowedDataActions;
+    private final List<KeyVaultDataAction> notDataActions;
 
     /**
-     * Creates a new {@link KeyVaultPermission} with the specified allowed and denied actions and data actions.
+     * Creates a new {@link KeyVaultPermission} with the specified allowed and forbidden actions and data actions.
      *
      * @param allowedActions The actions this {@link KeyVaultPermission permission} allows.
-     * @param deniedActions The actions this {@link KeyVaultPermission permission} denies.
+     * @param notActions The actions this {@link KeyVaultPermission permission} forbids.
      * @param allowedDataActions The data actions this {@link KeyVaultPermission permission} allows.
-     * @param deniedDataActions The data actions this {@link KeyVaultPermission permission} denies.
+     * @param notDataActions The data actions this {@link KeyVaultPermission permission} forbids.
      */
-    public KeyVaultPermission(List<String> allowedActions, List<String> deniedActions, List<String> allowedDataActions, List<String> deniedDataActions) {
+    public KeyVaultPermission(List<String> allowedActions, List<String> notActions,
```

Comment:
These should be `actions`, `notActions`, `dataActions`, and `notDataActions`. A decision was made in adparch review to keep them the same as swagger.